src/index.ts

```diff
@@ -7,6 +7,7 @@ import * as svelte from 'svelte/compiler';
 import * as chokidar from 'chokidar';
 import * as babel from '@babel/core';
 import * as glob from 'glob';
+import * as Terser from 'terser';
```

Comment:
Just a preference of mine 😅 
```suggestion
import * as terser from 'terser';
```

Reply:
ah, i did so because terser's README uses it like that (`var Terser = require('terser');`)

Reply:
Ahh i see in their readme, yeah... i think that's their chosen style 😅 

I usually keep all imports lowercase (except for actual classes) and i'm realizing now that i must not have the correct eslint rule applied to check for this.

Reply:
what should i put then?

Reply:
I'll change it at some point, it does not have to be done on this PR 👍